src/category_theory/natural_transformation.lean

```diff
@@ -2,6 +2,11 @@
 Copyright (c) 2017 Scott Morrison. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Tim Baumann, Stephen Morgan, Scott Morrison, Floris van Doorn
+-/
+import category_theory.functor
+
+/-!
+# Natural transformations
 
 Defines natural transformations between functors.
```

Comment:
Can you rapidly explain what natural transformations are?

Reply:
I'm of two minds about this (and the other similar comments below). My gut reaction is that these are all basic definitions that can be found in the first chapter or so of any text on category theory. I don't think that the mathlib docs should have to explicitly state basic definitions like these, unless there is something unusual about the formalization.

On the other hand, there is something to be said for convenience. More explanation here could save folks who are unfamiliar with category theory from having to dig around on Google. Maybe just some links to Wikipedia would work as a compromise?

Reply:
Just a quick recap is enough. This is a matter of no line vs three lines.

Reply:
Done.